website/source/guides/external/lxc.html.md

```diff
@@ -20,57 +20,17 @@ documentation.
 - Nomad [LXC][lxc-docs] external driver documentation
 - Nomad LXC external driver [repo][lxc-driver-repo]
 
-## Estimated Time to Complete
-
-20 minutes
-
-## Challenge
-
-You need to deploy a workload using [Linux Containers][linux-containers-home].
-Configure the client nodes that need to run this workload appropriately. You
-will also need to install the `lxc-templates` package which will provide the
-templates needed to start your containers.
-
-## Solution
-
-Install and configure the LXC external driver plugin. Verify the configuration
-on the client node and deploy the application.
-
-## Prerequisites
-
-To perform the tasks described in this guide, you need to have a Nomad
-environment with Consul installed. You can use this
-[repo](https://github.com/hashicorp/nomad/tree/master/terraform#provision-a-nomad-cluster-in-the-cloud)
-to easily provision a sandbox environment. This guide will assume a cluster with
-one server node and one client node.
-
--> **Please Note:** This guide is for demo purposes and is only using a single
-server node. In a production cluster, 3 or 5 server nodes are recommended.
-
 ## Steps
 
 ### Step 1: Verify Client Node Configuration
 
 External drivers must be placed in the [plugin_dir][plugin_dir] directory which
```

Comment:
Not sure what "verify" means, and when reading it in the preview this step doesn't seem to add much value.  Instead of this being step 1 could we turn into a